py/reporter_service.py

```diff
@@ -50,6 +51,24 @@ def serve_forever(self):
   def make_thread_locals(self):
     setattr(thread_local, 'segment_matcher', valhalla.SegmentMatcher())
 
+    levels = "0,1"
+    if os.environ.get('REPORT_LEVELS'):
+      levels = os.environ.get('REPORT_LEVELS')
+    setattr(thread_local, 'report_levels', set([ int(i) for i in levels.split(',')]))
+
+    trans_levels = "0,1"
+    if os.environ.get('TRANSITION_LEVELS'):
+      trans_levels = os.environ.get('TRANSITION_LEVELS')
+    setattr(thread_local, 'transition_levels', set([ int(i) for i in trans_levels.split(',')]))
+
+    threshold_sec = 15
+    if os.environ.get('THRESHOLD_SEC'):
+      threshold_sec = bool(strtobool(str(os.environ.get('THRESHOLD_SEC'))))
+    setattr(thread_local, 'threshold_sec', threshold_sec)
+
+    provider = os.environ.get('PROVIDER', '')
```

Comment:
since the reporter wont be sending these to the datastore this is no longer needed. we'll be doing it in the java code